lib/player.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Player
+  attr_reader :number, :name, :symbol
+  attr_accessor :wins
```

Comment:
You should use `attr_reader / attr_writer` instead of `attr_accessor`

Reply:
@Redvanisation Thank you. :+1: